tracee/tracee.go

```diff
@@ -407,8 +441,9 @@ func (t *Tracee) processFileWrites() {
 				continue
 			}
 
-			pathname := path.Join(t.config.OutputPath, strconv.Itoa(int(meta.MntID)))
-			if err := os.MkdirAll(pathname, 0755); err != nil {
+			pathname := path.Join(t.config.OutputPath, strconv.Itoa(int(meta.MntID)), "written-files")
+			ensureOutputDir(pathname)
```

Comment:
Why create a special (long) function for what os.MkdirAll does for "free"?

Reply:
my original thought was to be able to reuse directories while considering their permissions. mkdirall doesn't care about the permissions. but I'm not sure this is important anymore.